utide/robustfit.py

```diff
@@ -103,15 +103,15 @@ def welsch(r):
     return w
```

Comment:
@wesleybowman Based on what you said I think it is safe to remove `robustfit.py`, right?